zebra-chain/src/orchard/arbitrary.rs

```diff
@@ -41,8 +41,12 @@ impl Arbitrary for note::Nullifier {
     fn arbitrary_with(_args: Self::Parameters) -> Self::Strategy {
         use halo2::arithmetic::FieldExt;
 
-        (any::<u64>())
-            .prop_map(|number| Self::from(pallas::Scalar::from_u64(number).to_bytes()))
+        (vec(any::<u8>(), 64))
+            .prop_map(|bytes| {
+                let bytes = bytes.try_into().expect("array is the correct length");
```

Comment:
Oops, but now we have a CI cache
```suggestion
                let bytes = bytes.as_slice().try_into().expect("array is the correct length");
```